jasmine-expect/jasmine-expect.d.ts

```diff
@@ -0,0 +1,127 @@
+// Type definitions for jasmine-expect v2.0.0-beta1
+// Project: https://github.com/JamieMason/Jasmine-Matchers
+// Definitions by: Nicolas Gaignoux <https://github.com/PillowPillow>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/*
+Typings 2015 Nicolas Gaignoux
```

Comment:
License is here. https://github.com/borisyankov/DefinitelyTyped/blob/master/LICENSE
This is may not be explicitly shown in individual files.